docs/tutorial/intro.md

```diff
@@ -1,15 +1,6 @@
-# Tutorial
+# Introduction
 
 This section contains instructions and examples to get you started. It's written in tutorial style, intended to be read start to finish.
 
-@@@ index
-
 * [In-memory model](in-memory-model.md)
-* [DB-backed model](db-backed-model.md)
-* [Composite model](composite-model.md)
-
-@@@
-
-## Table of Contents
-
-@@toc { depth=2 }
+* [DB Backed model](in-memory-model.md)
```

Comment:
I think Laika has a comparable feature, might be useful.

https://typelevel.org/Laika/latest/07-reference/01-standard-directives.html#navigationtree

Reply:
For now I just want to get the site moved over. We can finesse things after that.